### PR TITLE
add: `yafetch-git`

### PR DESCRIPTION
**Pull request by Henryws**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [x] Add maintainer to pacscript

### CODE REVIEW

**Review by volitank — DISMISSED** [review]

File: packages/yafetch-git/yafetch-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name="yafetch-git"
+gives="yafetch"
+breaks="${gives}-deb ${gives}-bin ${gives}-app"
```
Comment (wizard-28) [inline packages/yafetch-git/yafetch-git.pacscript:3]:
```suggestion
breaks="${gives}-deb ${gives}-bin ${gives}-app ${gives}-git"
```

File: packages/yafetch-git/yafetch-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  sudo install -Dm755 yafetch "/usr/src/pacstall/yafetch-git/usr/bin/yafetch"
```
Comment (wizard-28) [inline packages/yafetch-git/yafetch-git.pacscript:21]: Use `STOWDIR`

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Aren't you going to contact the devs?

**Comment by Henryws** [comment]
> Aren't you going to contact the devs?

I am the dev!

**Comment by volitank** [comment]
Should we even include this? I heard there was a better python version coming soon.

**Comment by Henryws** [comment]
> Should we even include this? I heard there was a better python version coming soon.

I can open a separate PR for `yapfetch`

**Comment by wizard-28** [comment]
> > Aren't you going to contact the devs?
> 
> I am the dev!

Have you tried contacting yourself?

**Review by wizard-28 — APPROVED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Check CI logs: https://github.com/pacstall/pacstall-programs/runs/5258913812?check_suite_focus=true

**Review by volitank — APPROVED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**